### ISSUE TITLE
There is no tooltip that displays the full container name

### ISSUE THREAD

**Issue by Softvision-VladBacia** [mozilla/testpilot-containers#609]
**[Affected versions]:**
- Firefox 54
- Containers Experiment Build: 2.3.0

**[Affected Platforms]:**
- All Windows
- All Linux
- All Mac

**[Prerequisites]:**
- Containers Experiment is installed on a new browser profile.
- Multiple containers with a long name are already created.

**[Steps to reproduce]:**
1. Start the browser with the profile from prerequisites.
2. Hover the "+" open new tab button so the list of containers is displayed.
3. Move the mouse cursor over the containers with a long name.

**[Expected results]:**
- Full container name is displayed in a tooltip.

**[Actual results]:**
- Only a part of the name is displayed.

**[Notes]:**
- Attached a screenshot of the issue:
![no tooltips with the full name of the container](https://user-images.githubusercontent.com/9248513/27385920-d28816e4-569c-11e7-8546-1fd770a50eb6.jpg)

**Comment by kjozwiak** [comment]
We've talked about adding several tooltips throughout the containers test pilot in #112. It would definitely be useful if users could see the entire container name under the "+" button but I don't think this a blocker for the Shield Study release.

I renamed #112 and added this bug [here](https://github.com/mozilla/testpilot-containers/issues/112#issuecomment-310241334) as another component/area that's in need of a tooltip.

If we're fine tackling this in #112, we can just close this off.
